### ISSUE TITLE
double tap

### ISSUE THREAD

**Issue by FernandoMiguel** [fetlife/android#429]
could we make double tap on photos to love (a la instagram) instead of zoom?
nobody ever zooms, and i  rather use pinch  to zoom than to double tap.

also i use my hand with my right hand and I cant reach the love button anyway.

thanks in advance

**Comment by reneklacan** [comment]
+1

**Comment by kistitan** [comment]
Hi @FernandoMiguel ,

I use zoom personally, but only with the zoom gesture. So double tap zoom indeed does not make much sense, so adding some useful functionality would be a nice improvement.

Let us discuss the idea and see what we can do (probably rather mid term than short term(

**Comment by FernandoMiguel** [comment]
I understand you have your own planning.
but if there anything i can say to persuade the team to move that to a short term, would be lovely.

I love the app, and use it more than the site at the moment.
clicking the love button is very frustrating, as it misses most of the taps, and ends up closing the tab, requiring additional taps to restore it  and the click the love button....
all of this done with two hands instead of just one.

Thanks for your work!

**Comment by kistitan** [comment]
@FernandoMiguel ,

Yes, I do understand however it has some difficulties both at technical and at feature level.
For displaying the image we use an external library that does not give straightforward way to change it like that way.
Of course it is doable but gives some more complexity.

Also just turning on this feature automatically for everyone is not an option as users would complain about unwanted and unexpected loves. Of course it is also solvable as we could add setting in the options menu, notify the users with popup or toast message, but still it is another little extra thing to solve.

So we will get there, but will need some time.

Thank you for your understandings.
